textworld/generator/text_grammar.py

```diff
@@ -455,8 +430,8 @@ def get_all_expansions_for_type(self, type: str):
             All possible names.
         """
         expansions = self.get_all_expansions_for_tag("#({})#".format(type))
-        for room_type in self.grammar["#room_type#"]:
-            expansions += self.get_all_expansions_for_tag("#{}_({})#".format(room_type, type))
```

Comment:
@tavianator why removing the parentheses? They are needed to match rules like https://github.com/microsoft/TextWorld/blob/master/textworld/generator/data/text_grammars/house_obj.twg#L73

Reply:
Uh that's a very good question!  Will fix

Reply:
To be fair, I don't think that is used often. It is mostly there because we got asked many times what's the "complexity" of our text templates. However, trying to get a count of all the possible expansions takes too much time the way I implemented it (with recursion). There's probably a better way of doing that, though.